src/natcap/invest/coastal_vulnerability.py

```diff
@@ -462,6 +462,13 @@
                                     "type": "integer",
                                     "about": "Shore point ID"
                                 },
+                                "R_hab": {

```

Comment:
Moving this up so that the `r_hab` column is matched before we get to the `[HABITAT]` pattern, which matches everything.